### DISCUSSION TITLE
[Bug] Split view options are missing from the tab context menu in other tab extensions

### DISCUSSION THREAD

**Discussion by Zerodya** [Floorp-Projects/Floorp#661] [Feature Request]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

I use my own customized Tree Style Tab extension for vertical tabs instead of Floorp's native implementation because it's smoother and more polished. However these options are missing from the tab context menu: 
- Fixed in split view 
- Close split tab

May be because theese options have been hardcoded in the context menu of the native tab bar only? I have no idea.

### To Reproduce

1. Install Tree Style Tab add-on
2. Right click on tab inside Tree Style Tab
3. Find that Split view options are missing

### Expected behavior

Split view options show up in every tab context menu, not only the native one.

### Screenshots

https://github.com/Floorp-Projects/Floorp/assets/73220426/0c7507d8-defc-40a5-a12a-273144d9ff90

### OS Info

Artix Linux

### Floorp Version

11.7.1

**Comment by regularentropy** [comment]
> May be because theese options have been hardcoded in the context menu of the native tab bar only? I have no idea.

Yes. This behavior is applicable only to the native sidebar.

**Comment by Zerodya** [comment]
Would it be easy to allow those options to show up in other context menus? And if not, are there any plans to allow using the split view feature via alternative ways (for example tab drag-and-drop or other methods)?

This is a great browser and split view is the only feature I'm missing because I like to hide the horizontal tab bar, so it would be great to make the feature more accessible.

**Comment by Ryosuke-Asano** [comment]
Cannot do that...
